src/code/actions.js

```diff
@@ -390,6 +382,7 @@ export function submitDrake(targetDrakeIndex, userDrakeIndex, correct, incorrect
           label: "~BUTTON.TRY_AGAIN",
           action: incorrectAction || "dismissModalDialog"
         },
+        top: "65%"
```

Comment:
Some changes here to move dialogs out of the way and fix functionality...these messages should be gone by the next classroom test, so probably not worth too much thought.